client/network/src/discovery.rs

```diff
@@ -469,18 +458,24 @@ mod tests {
 						out,
 						secio,
 						endpoint,
-						libp2p::core::upgrade::Version::V1
-					)
+						upgrade::Version::V1
+					).map_ok(|(id, stream)| ((stream, id)))
 				})
-				.and_then(move |(peer_id, stream), endpoint| {
+				.and_then(move |(stream, peer_id), endpoint| {
```

Comment:
What's this about?

Reply:
Ah, I got a compilation error here and initially thought it was because the variables were in the wrong order.